source/_integrations/nuki.markdown

```diff
@@ -24,9 +24,26 @@ The Nuki integration allows you to control [Nuki Smart Locks](https://nuki.io/en
 ## Prerequisites
 
 To add a Nuki bridge to your installation, you need to enable developer mode on your bridge and define a port and an access token. This can be achieved using the [Android app](https://play.google.com/store/apps/details?id=io.nuki) or [iPhone app](https://apps.apple.com/app/nuki-smart-lock/id1044998081). Go to manage my devices, and select the bridge. Within the bridge configuration turn on the HTTP API and check the details in the screen. Please note that the API token should be 6-20 characters long, even though the app allows you to set a longer one.
+For faster updates you can configure to use the callback function of the Nuki bridge. This requires your Home Assistant to be reachable via HTTP by the Nuki bridge. If required, you can set a custom URL for the callback, which allows HTTP traffic to `/api/nuki/callback/{entry-id}`.
 
 {% include integrations/config_flow.md %}
 
+{% configuration_basic %}
+  Host:
+    description: Hostname or IP address of your Nuki bridge, e.g., `192.168.1.25`.
+  Port:
+    description: Port of the Nuki bridge HTTP API.
+    default: 8080
+  Token:
+    description: Token to authenticate against the Nuki bridge HTTP API.
+  "Enable Nuki callback":
+    description: Enable the callback function of the Nuki bridge HTTP API.
+    default: false
```

Comment:
Why does this need enabling?

Reply:
I thought to make this opt-in, because of the requirement of an http endpoint. I don't know how many people still have an internal http url configured, how do you think we could handle this?

Reply:
I'm not sure how that is related to all of that at all? Why does it matter?